### PR TITLE
Update Primitives3D.py

### PR DESCRIPTION
**Pull request by PipKat**
This one was in fairly rough shape--so lots of edits in its 869 lines.

### CODE REVIEW

**Comment by akaszynski** [comment]
Our build machine needs to be restarted or checked into as `pyansys.PyAEDT` is failing consistently.  Pinging @maxcapodi78.

File: pyaedt/modeler/Primitives3D.py [hunk 1]
```diff
@@ -5,45 +5,65 @@
...
-        dimensions_list :
-            list of dimensions of X, Y, Z
-        name :
-            box name. Optional, if nothing default name will be assigned
-        matname :
-            material name. Optional, if nothing or if invalid, default material will be assigned
+        position : list
+            Center point for the box in a list of [x, y, z] coordinates. 
+        dimensions_list : list
+           Dimensions for the box in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives3D.py:43]:
```suggestion
           Dimensions for the box in a list of ``[x, y, z]`` coordinates.
```

File: pyaedt/modeler/Primitives3D.py [hunk 2]
```diff
@@ -5,45 +5,65 @@
...
-        position :
-            ApplicationName.modeler.Position(x,y,z) object
-        dimensions_list :
-            list of dimensions of X, Y, Z
-        name :
-            box name. Optional, if nothing default name will be assigned
-        matname :
-            material name. Optional, if nothing or if invalid, default material will be assigned
+        position : list
+            Center point for the box in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives3D.py:41]:
```suggestion
            Center point for the box in a list of ``[x, y, z]`` coordinates. 
```

File: pyaedt/modeler/Primitives3D.py [hunk 3]
```diff
@@ -479,25 +549,27 @@ def convert_segments_to_line(self, object_name):
...
-            dimension list
-        name :
-            Object Name (Default value = None)
-        matname :
-            material name. Optional, if nothing default material will be assigned
-
+        cs_plane :
+            Coordinate system plane for orienting the rectangle. 
+        position : list
+            Center point of the rectangle in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives3D.py:559]:
```suggestion
            Center point of the rectangle in a list of ``[x, y, z]`` coordinates. 
```

File: pyaedt/modeler/Primitives3D.py [hunk 4]
```diff
@@ -479,25 +549,27 @@ def convert_segments_to_line(self, object_name):
...
-            Object Name (Default value = None)
-        matname :
-            material name. Optional, if nothing default material will be assigned
-
+        cs_plane :
+            Coordinate system plane for orienting the rectangle. 
+        position : list
+            Center point of the rectangle in a list of [x, y, z] coordinates. 
+        dblList : list
+            Dimensions of the rectangle in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives3D.py:561]:
```suggestion
            Dimensions of the rectangle in a list of ``[x, y, z]`` coordinates.
```

File: pyaedt/modeler/Primitives3D.py [hunk 5]
```diff
@@ -594,29 +664,31 @@ def create_udm(self, udmfullname, udm_params_list, udm_library='syslib'):
...
-        height :
-            height
-        name :
-            Object Name (Default value = None)
-        matname :
-            material name. Optional, if nothing default material will be assigned
+        cs_axis :
+            Coordinate system axis for orienting the cone. 
+        position : list
+            Center point of the cone's base circle in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives3D.py:674]:
```suggestion
            Center point of the cone's base circle in a list of ``[x, y, z]`` coordinates. 
```

File: pyaedt/modeler/Primitives3D.py [hunk 6]
```diff
@@ -594,29 +664,31 @@ def create_udm(self, udmfullname, udm_params_list, udm_library='syslib'):
...
+        cs_axis :
+            Coordinate system axis for orienting the cone. 
+        position : list
+            Center point of the cone's base circle in a list of [x, y, z] coordinates. 
+        bottom_radius : float
+            Radius of the cone's base circle.
+        top_radius : float
+            Radius of the cone's top circle.
+        height : list
+            Height of the cone in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives3D.py:680]:
```suggestion
            Height of the cone in a list of ``[x, y, z]`` coordinates.
```

**Review by akaszynski — APPROVED** [review]
Approved due to passing build.  Thanks for reviewing this.

**Pull request merged**